admin_guide/encrypting_data.adoc

```diff
@@ -0,0 +1,261 @@
+[[admin-guide-encrypting-data-rest]]
+= Encrypting Data at Rest
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[encrypting-data-overview]]
+== Overview
+
+This topic shows how to enable and configure encryption of secret data at rest.
+
+[[encrypting-data-configuration]]
+== Configuration and Determining Whether Encryption at Rest is Already Enabled
+
+In order to activate experimental support for data encryption, an argument
+*--experimental-encryption-provider-config* should be passed to the Kubernetes API server:
+
+.Excerpt of *_master-config.yaml_*
+====
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  apiServerArguments:
+    experimental-encryption-provider-config:
+    - /path/to/encryption.config
+----
+====
+
+[[encrypting-data-encryption-config]]
+== Understanding the Encryption at Rest Configuration
+
+.Encryption configuration file with all available providers
+====
+
+[source,yaml]
+----
+kind: EncryptionConfig
+apiVersion: v1
+resources:
+  - resources:
+    - secrets
+    providers:
+    - identity: {}
+    - aesgcm:
```

Comment:
The table below probably works better than <1>. Maybe mention "See the table below for descriptions of the parameters in the following sample code."